api/v1beta1/nodemaintenance_webhook.go

```diff
@@ -241,13 +241,3 @@ func getNode(nodeName string, client client.Client) (*v1.Node, error) {
 	}
 	return &node, nil
 }
-
```

Comment:
IIUC you remove here a check for the master role, which I am not sure when it will be deprecated...

Reply:
it's been replaced by `nodes.IsControlPlane(node)` from common repo
/lgtm 
just to be sure I didn't get it wrong
/hold